Code/EnginePlugins/OpenXRPlugin/OpenXRDeclarations.h

```diff
@@ -0,0 +1,56 @@
+#pragma once
+
+#define XR_LOG(exp)                                                                                              \
+  do                                                                                                                   \
+  {                                                                                                                    \
+    auto s = (exp);                                                                                                   \
+    if (s != XR_SUCCESS)                                                                                               \
+    {                                                                                                                  \
+      ezLog::Error("OpenXR failure: {0}:{1} {2}", __FILE__, __LINE__, s);                                                     \
+    }                                                                                                                  \
+  } while (false)
+
+#define XR_CHECK_LOG(exp)                                                                                              \
```

Comment:
Should be named SUCCEED_OR_RETURN in some way. Maybe check whether all these different macros are needed.

Reply:
Cleaning up log functions a bit.